### ISSUE TITLE
No Widget

### ISSUE THREAD

**Issue by dinosaw** [VxJasonxV/twitch-stream-status#1]
Hello. I've installed this plugin on my WordPress site successfully, however when I go into "Widgets" to add this to my sidebar, I don't see any widgets relating to this / Twitch. Is this still being developed or am I missing something?

**Comment by VxJasonxV** [comment]
Have you already configured casters in Settings > Twitch? Are any of them Live? I think my "only show the widget if there's content to show" logic also prevents it from even showing up in the Sidebar list.

**Comment by dinosaw** [comment]
Thanks for the quick response. That would indeed be the problem - the casters I had added were all offline. However I just added a known online streamer, and the widget appeared.

I agree that it is a good idea to prevent the widget from being shown on the actual site, however it is a bit odd for the actual widget in the Dashboard to not appear. Essentially that means you cannot move / edit the Widget until someone you have added is online. But that is just my two cents! It's working and I understand how it works now, so thanks a ton for your work!
